src/base/Pool.sol

```diff
@@ -433,9 +435,7 @@ abstract contract Pool is Clone, ReentrancyGuard, Multicall, IPool {
         );
 
         // burn required number of ajna tokens to take quote from reserves
-        IERC20(_getArgAddress(AJNA_ADDRESS)).safeTransferFrom(msg.sender, address(this), ajnaRequired);
-
-        IERC20Token(_getArgAddress(AJNA_ADDRESS)).burn(ajnaRequired);
```

Comment:
`IERC20Token` import can be removed

Reply:
hey kirill, do we need to concern about non-standard bridged token? or this can be note to deployer to run integration test before deploying it.

Reply:
Since AJNA token is already circulating, it should be quite easy to test all success paths on a live L2 deployment with bridged quote, collateral and bwAJNA tokens

Reply:
I ran on Base yesterday, and Arbitrum, Optimism, and Polygon PoS today.  Renamed variables to be less Base-specific, even though token address is hardcoded in the test.